src/components/Collections/TokenDetail/index.tsx

```diff
@@ -16,13 +26,15 @@ import { MinterButton, MinterMenuButton, MinterMenuItem } from '../../common';
 import { TransferTokenModal } from '../../common/TransferToken';
 import { SellTokenButton, CancelTokenSaleButton } from '../../common/SellToken';
 import { BuyTokenButton } from '../../common/BuyToken';
-import { ipfsUriToGatewayUrl, uriToCid } from '../../../lib/util/ipfs';
+import { ipfsUriToGatewayUrl } from '../../../lib/util/ipfs';
 import { useSelector, useDispatch } from '../../../reducer';
 import {
   getContractNftsQuery,
   getNftAssetContractQuery
 } from '../../../reducer/async/queries';
 
+import fullscreen from '../../common/assets/fullscreen.svg';
```

Comment:
Instead of pulling this SVG, we've been using `react-feather` (https://feathericons.com/). It looks like `Maximize2` should work. Can you please change to that.

Reply:
Okay, I couldn't find any so I just went with the SVG.